Aztec/Classes/Libxml2/DOM/Data/ElementNode.swift

```diff
@@ -199,6 +201,27 @@ extension Libxml2 {
             return children.filter(filter).last
         }
 
+
+        /// Indicates whether the children of the specified node can be merged in, or not.
+        ///
+        /// - Parameters:
+        ///     - node: Target node for which we'll determine Merge-ability status.
+        ///
+        /// - Returns: true if both nodes can be merged, or not.
+        ///
+        func canMergeChildren(of node: ElementNode) -> Bool {
+            guard name == node.name && Set(attributes) == Set(node.attributes) else {
+                return false
+            }
+
+            guard let standardName = self.standardName else {
```

Comment:
I don't think the merge should fail just because the element has no standard name.

The truth is that the code below this point will need to be changed, so if you prefer to avoid analyzing this further now, we can proceed as it is and analyze in a second PR.